tests/tests_pytorch/models/test_tpu.py

```diff
@@ -259,12 +259,15 @@ def test_accelerator_set_when_using_tpu(devices):
 
 @pytest.mark.parametrize(
     ["cli_args", "expected"],
-    [("--tpu_cores=8", {"tpu_cores": 8}), ("--tpu_cores=1,", {"tpu_cores": "1,"})],
+    [
+        ("--accelerator=tpu --devices=8", {"accelerator": "tpu", "devices": 8}),
```

Comment:
Standalone tests seem to have an issue with the whitespace here between the flags.

You can try to fix the space escaping in the script
or use `pytest.param(..., id="foo")`

Reply:
Interesting, how did you find this? All I saw in the logs was "stopping because of potential error".

Reply:
It says:

```py
ERROR: not found: /__w/2/s/tests/tests_pytorch/models/test_tpu.py::test_tpu_devices_with_argparse[--accelerator=tpu
(no name '/__w/2/s/tests/tests_pytorch/models/test_tpu.py::test_tpu_devices_with_argparse[--accelerator=tpu' in any of [<DoctestModule tests/tests_pytorch/models/test_tpu.py>, <Module tests/tests_pytorch/models/test_tpu.py>])
```